### PR TITLE
Use consistent media colour in most popular container

### PR DESCRIPTION
**Pull request by robertberry-zz**
Also `dev-build` was screwed. Dunno who did that, but this fixes it.

screen shot 2015-05-20 at 16 20 55

### CODE REVIEW

**Comment by robertberry** [comment]
Fixes https://github.com/guardian/frontend/issues/9053

CC @sammorrisdesign

File: dev-build/conf/routes [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 GET         /humans.txt                                                                                              dev.DevAssetsController.humans()
 GET         /surveys/*file                                                                                           dev.DevAssetsController.surveys(file)
-GET         /commercial/test-page                                                                                    controllers.commercial.CreativeTestPage.allComponents
+GET         /commercial/test-page                                                                                    controllers.commercial.CreativeTestPage.allComponents(k: List[String])
```
Comment (sammorrisdesign) [inline dev-build/conf/routes]: lol what

**Comment by sndrs** [comment]
:+1: spacing looks a bit tight?

File: dev-build/conf/routes (hunk 1 above)
Comment (robertberry) [inline dev-build/conf/routes]: Someone did it on the commercial app but not in dev-build ...

It basically looks for a k param in the query string. So if you go ?k[]=lol&k[]=dongs it'd pass List("lol", "dongs")

File: static/src/stylesheets/module/_headline-list.scss [hunk 2]
```diff
@@ -61,6 +61,11 @@
         fill: colour(neutral-2);
     }
 
+    .inline-camera,
+    .inline-volume-high {
+        fill: colour(media-default);
```
Comment (sammorrisdesign) [inline static/src/stylesheets/module/_headline-list.scss]: Why is video separate from this?

Comment (robertberry) [inline static/src/stylesheets/module/_headline-list.scss]: The colour is baked into that svg for some reason

Comment (sndrs) [inline static/src/stylesheets/module/_headline-list.scss]: should remove the colour if it is, seems like an oversight

**Pull request merged**